Examples/Algorithms/Geant4/src/SensitiveSteppingAction.cpp

```diff
@@ -109,9 +109,14 @@ void ActsExamples::SensitiveSteppingAction::UserSteppingAction(
     // Retrieve the event data registry
     auto& eventData = EventStoreRegistry::eventData();
 
+    if (eventData.particleHitCount.count(particleID) == 0) {
+      eventData.particleHitCount[particleID] = 0;
+    } else {
+      eventData.particleHitCount[particleID]++;
+    }
```

Comment:
I also thought about this, but was not sure if the `std::size_t` default constructed (random memory), or value constructed (as `0`)?
Also, the hit index would start at `1` then, right? Of course this would be easy solvable

Reply:
Okay, according to https://en.cppreference.com/w/cpp/container/unordered_map/operator_at it gets value constructed, so your solution would be possible

Reply:
yeah I am pretty sure that you get a default constructed object here

ah it is the hit index right. I thought it is the hit count. hmmm I guess we could default construct it to -1 but seems bad

however feel free to merge as is

Reply:
no we can just subtract 1 when constructing the `ActsFatras::Hit`, thats an easy solution to this

Reply:
haha true. then the name hit count also makes more sense